src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Dns.cs

```diff
@@ -9,23 +9,21 @@ namespace System.Net.NetworkInformation
 {
     internal static partial class StringParsingHelpers
     {
-        internal static string ParseDnsSuffixFromResolvConfFile(string filePath)
+        internal static string ParseDnsSuffixFromResolvConfFile(string data)
         {
-            string data = File.ReadAllText(filePath);
             RowConfigReader rcr = new RowConfigReader(data);
             string dnsSuffix;
 
             return rcr.TryGetNextValue("search", out dnsSuffix) ? dnsSuffix : string.Empty;
         }
 
-        internal static List<IPAddress> ParseDnsAddressesFromResolvConfFile(string filePath)
+        internal static List<IPAddress> ParseDnsAddressesFromResolvConfFile(string data)
```

Comment:
Why did you change these two methods to move the File.ReadAllText to the call site?  It looks like all call sites do it now?

Reply:
Ah, I see, to avoid reading the file twice in one place where we pass the same text to two different parsers. Ok.